.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: build-linux
+on: [push]
+#  schedule:
```

Comment:
Note for reviewer: left this like this pending discussion on whether these should be rebuilt on push, on schedule, or if a GitHub action is wanted at all.

Reply:
Will this only create the PR if the binaries have changed?

Reply:
The PR action only happens if the binaries _differ_, but I'm not sure if SDL builds are reproducible or not in the first place. (They might be, but I haven't checked) Either way, this workflow as-is will build from the tip of SDL master, so chances are in practice there will always be a commit or two that'll have happened to make the builds differ anyway.

Reply:
(The way I have this set up in other projects I contribute to is have it open a PR, e.g. "sync translations", let the action continually update it on a schedule, and periodically merge the PR in when sufficient changes have accumulated.)

Reply:
That sounds like PRs would build up over time? One for each commit/PR?

Reply:
Since it force-pushes to the same branch, there's only one PR that gets updated. Here's [an example](https://github.com/DMOJ/online-judge/pull/1256) from [this workflow](https://github.com/DMOJ/online-judge/blob/master/.github/workflows/updatemessages.yml).